zgrep: add page (#4131)
tldr-pages/tldr: pages/linux/zgrep.md

# zgrep

> > Grep pattern(s) of text (strings/regex) from files within compressed file (equivalent to grep -Z).

Reviewer: ```suggestion
> Grep text patterns from files within compressed file (equivalent to grep -Z).
```

Author: changed

> > Syntax: zgrep [grep_options] pattern file.

Reviewer: ```suggestion
```
This is redundant, as you can see it in the examples. We omit that in tldr pages.

Author: removed

> - Grep pattern from compressed file (case-sensitive):

Reviewer: ```suggestion
- Grep a pattern in a compressed file (case-sensitive):
```
Should be similar to this in other examples, too.

Author: changed

> `zgrep pattern path/to/compressed/file`

Reviewer: ```suggestion
`zgrep {{pattern}} {{path/to/compressed_file}}`
```
Missing braces for variable arguments. Should be similar in other examples too.

Author: changed, only line I wasnt sure about was line 23, I asked in gitter/tldr room. But used my best judgement, please check line 23

- Grep pattern from compressed file (case-insensitive):

> `zgrep -i pattern path/to/compressed/file`

Reviewer 2: need braces here
